### PR TITLE
load config yaml manually, remove more references to static config

### PR DESCRIPTION
**Pull request by jesopo**

### CODE REVIEW

File: test/integration/utilsTest.ts [hunk 1]
```diff
@@ -24,7 +26,7 @@ describe("Test: utils", function() {
         const ourHomeserver = new UserID(await this.mjolnir.client.getUserId()).domain;
         assert.equal(
             out.formatted_body,
-            `it's fun here in <a href="https://matrix.to/#/${config.managementRoom}?via=${ourHomeserver}">${config.managementRoom}</a>`
+            `it's fun here in <a href="https://matrix.to/#/${managementRoomAlias}?via=${ourHomeserver}">${managementRoomAlias}</a>`
```
Comment (Gnuxie) [inline test/integration/utilsTest.ts:29]: Hmm I don't think we're making these urls properly https://spec.matrix.org/v1.1/appendices/#matrixto-navigation I don't think it makes sense to inject `via` for a room with an alias. I also don't think we should depend on management rooms having an alias (at least outside of tests, this is a test).

File: test/commands/UnbanBanCommandTest.ts [hunk 2]
```diff
@@ -18,10 +18,11 @@ import * as expect from "expect";
 import { Mjolnir } from "../../src/Mjolnir";
 import { DEFAULT_LIST_EVENT_TYPE } from "../../src/commands/SetDefaultBanListCommand";
 import { parseArguments } from "../../src/commands/UnbanBanCommand";
-import config from "../../src/config";
+import { read as config_read } from "../../src/config";
 import { RULE_ROOM, RULE_SERVER, RULE_USER } from "../../src/models/ListRule";
 
 function createTestMjolnir(defaultShortcode: string|null = null): Mjolnir {
+    const config = config_read();
```
Comment (Gnuxie) [inline test/commands/UnbanBanCommandTest.ts:25]: hmm wait why's this named with an underscore?

Comment (jesopo) [reply test/commands/UnbanBanCommandTest.ts:25]: oh whoops. should be camel case

File: test/integration/utilsTest.ts (hunk 1 above)
Comment (jesopo) [reply test/integration/utilsTest.ts:29]: can this be addressed in a followup PR?

**Review by Gnuxie — APPROVED** [review]

**Pull request merged**